functionsassignment.js

```diff
@@ -16,3 +16,5 @@ fName='Somya';
 lName='Kakarla';      
```

Comment:
Where is the code for first question?

Reply:
Both questions execution done  in one code itself sir